algosdk/v2client/indexer.py

```diff
@@ -152,9 +152,9 @@ def accounts(
             query["limit"] = limit
         if next_page:
             query["next"] = next_page
-        if min_balance:
+        if min_balance is not None and min_balance >= 0:
```

Comment:
The second part of the conditional (>=0) seems to change the existing behavior of the request. Before, a negative param would return an error from the server, but now the client just ignores the negative. 

I don't feel too strongly about it, but I think returning an error (either client or server side) seems to be a bit more desirable.